src/main/java/PlateSensor.java

```diff
@@ -1,5 +1,7 @@
-package java;
+package main.java;
 
-public class PlateSensor {
-   
+public class PlateSensor extends Sensor {
+	public PlateSensor() {
+		status = 0;
```

Comment:
what is the "0" status?
I recommend to use constants.

Reply:
Constants implemented.